tox.ini

```diff
@@ -77,7 +77,7 @@ extras =  # architectures used in the package tests
     soap-bpnn
     pet
 allowlist_externals = bash
-commands_pre = bash {toxinidir}/tests/resources/generate-outputs.sh
+; commands_pre = bash {toxinidir}/tests/resources/generate-outputs.sh
```

Comment:
Maybe we find a way at some point to cache this script... I also always comment this line and forget about it.

Reply:
😅 At least CI finds about it!